src/deploy/jenkins-uat.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('node-fetch');
+
+module.exports = () => {
+    const jenkinsUrl = "https://ctv-jenkins-02.truex.com/generic-webhook-trigger/invoke";
+    return fetch(jenkinsUrl, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+            "token": "TAR-HTML5_UAT",
+        },
+        body: JSON.stringify({
+            "whakapapaBranchName": "develop",
```

Comment:
this needs to be parameterized, probably make an arg to the function, since the host repo knows the travis settings, not this shared function as such